### PR TITLE
Rails 5 support

### PR DESCRIPTION
**Pull request by webhoernchen**
- Support for Rails 5 added
- Deprecation Warning for before_filter in Rails 5 fixed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@ pkg/*
 *.gem
 .bundle
 gemfiles/*.lock
+
```
Comment (sikachu) [inline .gitignore]: Do you mind remove changes to this `.gitignore?`

**Review by sikachu — CHANGES_REQUESTED** [review]
There's one thing that needs to be fixed.

Would you mind also rebasing against `master` as I just pushed some commits to fix Travis build? Thanks!

File: .gitignore (hunk 1 above)
Comment (webhoernchen) [inline .gitignore]: changes are removed

**Review by sikachu — APPROVED** [review]

**Comment by sikachu** [comment]
Thank you for your patch!

**Pull request merged**